ansible/roles/iroha-docker/templates/docker-compose.yml.j2

```diff
@@ -62,19 +62,21 @@ services:
 {% endfor %}
 {% endif %}
     volumes:
-      - psql_storage-{{ iroha_inventory_human_hostname }}:/var/lib/postgresql/data
+      - {{ iroha_volume_dir }}psql_storage-{{ iroha_inventory_human_hostname }}:/var/lib/postgresql/data
```

Comment:
You maybe have to use conditional and "path" filter.

Reply:
There are no `path` filter I used `dirname`, and put it to top of file